### PR TITLE
Feature/config-files

### PR DESCRIPTION
**Pull request by tiagojpdias**
- Split config files resembling Laravel's way
- Add `start` script to npm
- Send messages directly to author instead of replying to channel
- Delete the trigger message to avoid clutter
- Add shrug tag #14

### CODE REVIEW

File: config/index.js [hunk 1]
```diff
@@ -1,18 +1,30 @@
...
+}
+
+function loadModules(modulePath) {
+  const files =fs.readdirSync(modulePath);
+  const filesToLoad = files.filter(file => !file.startsWith('index'));
+
+  filesToLoad.forEach(file => {
+    const [fileName] = file.split('.');
+
+    // eslint-disable-next-line import/no-dynamic-require
```
Comment (quetzyg) [inline config/index.js:22]: What is this witchcraft?

File: package.json [hunk 2]
```diff
@@ -4,6 +4,7 @@
   "description": "",
   "main": "app.js",
   "scripts": {
+    "start": "node src/app.js",
```
Comment (quetzyg) [inline package.json]: #24 already has this change.

File: src/events/message.js [hunk 3]
```diff
@@ -5,35 +5,37 @@ const { allowedTags } = require('../boot');
...
 
-      author.send('**FODA-SE PÁ, NÃO PERCEBESTE A MENSAGEM CRL?!?!**', {
-        files: [attachment],
-      });
+  if (retryCount >= retryThreshold * 2) {
+    const attachment = new Attachment(getReservedImage('angryPepe'));
 
-      return;
-    }
+    author.send('**FODA-SE PÁ, NÃO PERCEBESTE A MENSAGEM CRL?!?!**', {
```
Comment (quetzyg) [inline src/events/message.js:16]: Probably best to merge this one first, or #24 and then come back to it, since some code overlaps.

**Review by quetzyg — CHANGES_REQUESTED** [review]
Left some comments

File: config/index.js (hunk 1 above)
Comment (tiagojpdias) [reply config/index.js:22]: Tells ESLint to look away on the next line.
By default ESLint's rules squawks when we use `require` elsewhere other than the top of file.

File: src/events/message.js (hunk 3 above)
Comment (tiagojpdias) [reply src/events/message.js:16]: Yep, after merging one of them we'll probably have conflicts.

**Review by zambinij — APPROVED** [review]

**Pull request merged**